philology/src/main/java/com/jcminarro/philology/PhilologyResources.kt

```diff
@@ -29,4 +47,109 @@ internal class PhilologyResources(baseResources: Resources) :
     override fun getStringArray(id: Int): Array<String> = resourcesUtil.getStringArray(id)
 
     override fun getTextArray(id: Int): Array<CharSequence> = resourcesUtil.getTextArray(id)
+
+    override fun getAnimation(id: Int): XmlResourceParser = baseResources.getAnimation(id)
```

Comment:
Should we keep the full method signature? I mean, parent's methods notify they could throw exceptions, would be good to have them annotated with the exception it could throw.
`@Throws(Resources.NotFoundException::class)`